access/grpc/client.go

```diff
@@ -40,7 +40,7 @@ const EmulatorHost = "127.0.0.1:3569"
 const TestnetHost = "access.devnet.nodes.onflow.org:9000"
 const CanarynetHost = "access.canary.nodes.onflow.org:9000"
 const MainnetHost = "access.mainnet.nodes.onflow.org:9000"
-const CrescendoHost = "access.crescendo.nodes.onflow.org:9000"
+const PreviewnetHost = "access.previewnet.nodes.onflow.org:9000"
```

Comment:
Once this gets deployed it will break until this hostname has been created. Would recommend waiting to release a new SDK version until after the domain is up and working

Reply:
Sounds good, we can delay the release until tomorrow morning.